specs/example.test.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert');
+const httpServer = require('http-server');
+const percySnapshot = require('@percy/webdriverio');
+
+async function createTodo() {
+  let newTodo = await $('.new-todo');
+  newTodo.setValue('New fancy todo');
+  await browser.keys('Enter');
+  // needed or the todo is never submitted
+  await browser.execute(() => document.querySelector('.new-todo').blur());
+}
+
+describe('example page', function () {
+  const PORT = 8000;
+  const TEST_URL = `http://localhost:${PORT}`;
+
+  let server = null;
+
+  before(() => {
+    // Start local server to host app under test.
+    server = httpServer.createServer({ root: `${__dirname}/../` });
+    server.listen(PORT);
+  });
+
+  after(() => {
+    // Shut down the HTTP server.
+    server.close();
+  });
+
+  it('Loads the app', async function () {
+    await browser.url(TEST_URL);
+
+    assert.equal(await browser.getTitle(), 'VanillaJS • TodoMVC');
```

Comment:
Any reason for `assert.equal` here but `expect().toEqual` in the other tests?

Reply:
Hahah no, that was the left over test